docs/libraries/workflow/GLOSSARY.md

```diff
@@ -0,0 +1,56 @@
+TODO
+
+# Glossary
+
+## Worker
+
+A process that's running workflows.
+
+There are usually multiple workers running at the same time.
+
+## Workflow
+
+A series of activies to be ran together.
+
+The code defining a workflow only specifies what activites to be ran. There is no complex logic (e.g. database queries) running within workflows.
```

Comment:
```suggestion
The code defining a workflow only specifies what activities can be ran. There is no complex logic (e.g. database queries) running within workflows.
```

> There is no complex logic (e.g. database queries) running within workflows.

I'm curious about the definition of "complex logic". Do we mean specifically code that mutates databases or hits APIs, or code that might be computationally complex?

Also does this sentence imply that "complex logic" doesn't run in any of Workflows -> Activities, or that *we* didn't add any "complex logic" when building this notion of what Workflows are?